pkg/resource/typed/types_resource_test.go

```diff
@@ -0,0 +1,78 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package typed_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/cosi-project/runtime/pkg/resource"
+	"github.com/cosi-project/runtime/pkg/resource/meta"
+	"github.com/cosi-project/runtime/pkg/resource/typed"
+)
+
+type TestSpec struct {
+	Var int
+}
+
+func (t TestSpec) DeepCopy() TestSpec {
+	return t
+}
+
+type Test = typed.TypedResource[TestSpec, TestRD]
```

Comment:
this feels like it should be `typed.Resource`

Reply:
Done!